docs/CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Added
+-----
+- Add ``run_process`` method to ``Process`` to support triggering of a
```

Comment:
Mention that this applies for python processes - here and also in the commit message.